specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -1019,9 +1019,11 @@ components:
         verkregenNamensSamenwerkingsverband:
           $ref: "#/components/schemas/Waardelijst"
           description: "Mogelijke waarden zijn te vinden in deze [Waardelijst](http://www.kadaster.nl/schemas/waardelijsten/Samenwerkingsverband/)"
-        aantekening:
-          $ref: "#/components/schemas/Aantekening"
-          description: "Een aantekening is een verwijzing naar een ter inschrijving aangeboden stuk. Een aantekening op een tenaamstelling van een zakelijk recht is meestal een beperking. Bijvoorbeeld de verkrijging van een aandeel in een zakelijk recht onder opschortende voorwaarde, een beperking van de handelingsbevoegdheid van de zakelijk gerechtigde, of een koopovereenkomst"
+        aantekeningen:
+          type: array
+          items:
+            $ref: "#/components/schemas/Aantekening"
+            description: "Een aantekening is een verwijzing naar een ter inschrijving aangeboden stuk. Een aantekening op een tenaamstelling van een zakelijk recht is meestal een beperking. Bijvoorbeeld de verkrijging van een aandeel in een zakelijk recht onder opschortende voorwaarde, een beperking van de handelingsbevoegdheid van de zakelijk gerechtigde, of een koopovereenkomst"
```

Comment:
als je die description nu een niveau hoger zet, wordt die wel getoond en levert geen waarschuwing in SwaggerHub